charmhelpers/contrib/storage/linux/ceph.py

```diff
@@ -1865,6 +1880,7 @@ def add_op_create_replicated_pool(self, name, replica_count=3, pg_num=None,
             'name': name,
             'replicas': replica_count,
             'pg_num': pg_num,
+            'crush-profile': crush_profile
```

Comment:
Does there need to be a default here (or is this now covered by my other comment above?)

Reply:
I think this one is resolved with the suggestion above